pom.xml

```diff
@@ -23,8 +25,15 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.17</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```

Comment:
we need not use alfa version of dependencies.

Reply:
done